test/unit/spec/nbextensions/appCell2/widgets/appCellWidget-spec.js

```diff
@@ -205,7 +323,6 @@ define([
     }
 
     async function startRunningCell(ctx) {
-        cellStartUp(ctx);
```

Comment:
remove the cell start up from this function and turn it into a shortcut for the cell start up steps.